doc/terminal-v2-roadmap.md

```diff
@@ -0,0 +1,76 @@
+# Terminal 2.0 Roadmap
+
+## Overview
+
+This document outlines the roadmap towards delivering Windows Terminal 2.0 by spring 2021.
+
+## Milestones
+
+The Windows Terminal project is engineered and delivered as a set of 4-week milestones. New features will go into [Windows Terminal Preview](https://aka.ms/terminal-preview) first, then a month after they been in Preview, those features will move into [Windows Terminal](https://aka.ms/terminal).
+
+| Duration | Activity | Releases |
+| --- | --- | --- |
+| 2 weeks | Dev Work<br/> <ul><li>Fixes / Features for future Windows Releases</li><li>Fixes / Features for Windows Terminal</li></ul> |  Release to Internal Selfhosters at end of week 2 |
+| 1 week | Quality & Stability<br/> <ul><li>Bug Fixes</li><li>Perf & Stability</li><li>UI Polish</li><li>Tests</li><li>etc.</li></ul>| Push to Microsoft Store at end of week 3 |
+| 1 week | Release <br/> <ul><li>Available from [Microsoft Store](https://aka.ms/terminal) & [GitHub Releases](https://github.com/microsoft/terminal/releases)</li><li>Release Notes & Announcement Blog published</li><li>Engineering System Maintenance</li><li>Community Engagement</li><li>Docs</li><li>Future Milestone Planning</li></ul> | Release available from Microsoft Store & GitHub Releases |
+
+## Terminal Roadmap / Timeline
+
+| Milestone end date | Milestone Name | Preview Release Blog Post |
+| ------------------ | -------------- | ------------------------- |
+| 2020-06-30 | [1.1](https://github.com/microsoft/terminal/milestone/24) in Windows Terminal Preview |

```

Comment:
nit: maybe add a note that "the specific end dates are subject to change / are a rough estimate"